atom/browser/api/atom_api_app.cc

```diff
@@ -183,7 +187,7 @@ void App::OnFinishLaunching() {
   auto browser_context = static_cast<AtomBrowserContext*>(
       AtomBrowserMainParts::Get()->browser_context());
   auto handle = Session::CreateFrom(isolate(), browser_context);
-  default_session_.Reset(isolate(), handle.ToV8());
+  g_wrap_session.Run(handle.ToV8());
```

Comment:
The `g_wrap_session` should be called in `Session::CreateFrom`, and `_setWrapSession` should be implemented in `atom_api_session.cc` too, so we don't have to call it for every place that creates a session.

Also I still prefer implementing `DefaultSession` in C++, it would be very helpful when we need to get it in C++ code.